source/atlas-data-lake-testing/tests/getMore.yml

```diff
@@ -0,0 +1,33 @@
+collection_name: &collection_name "driverdata"
+database_name: &database_name "test"
+
+tests:
+  -
+    description: "A successful find event with getMore"
+    operations:
+      -
+        object: collection
+        name: find
+        arguments:
+          filter: { a: { $gte: 2 }}
+          sort: { a: 1 }
+          batchSize: 3
+          limit: 4
+    expectations:
+      -
+        command_started_event:
+          command:
+            find: *collection_name
+            filter: { a: { $gte : 2 }}
+            sort: { a: 1 }
+            batchSize: 3
+            limit: 4
+          command_name: "find"
+          database_name: *database_name
+      -
+        command_started_event:
+          command:
+            batchSize: 1
```

Comment:
Let's add a comment here.

```
# Assertions for "getMore" and "collection" fields are omitted, as they will have arbitrary values
# TODO: This can be changed to use wildcard matchers when SPEC-1215 is implemented
```